REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
I'm looking into it.  Superficially it doesn't look as easy as using DualNumbers.  I think they didn't intend for anyone to interact with their data types except through the ForwardDiff API, but we have to in order to use our special data structures.   It would be really nice if we could, though, for many reasons.